.github/workflows/ci.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
             # One test at a time to address ballooning memory.
             # https://github.com/illinois-ceesd/mirgecom/issues/212
             for filename in test_*.py ; do
-              python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./"$filename"
+              python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./"$filename" || exit $?
```

Comment:
> > * AFAIK, Github CI runs its scripts with `set -e` by default.
> 
> [Yes](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell), but we override it.
> 
> Edit: here is where we override that:
> 
> https://github.com/illinois-ceesd/mirgecom/blob/d5c6b0ddf8da1778268e13df169174216a5aef3c/.github/workflows/ci.yaml#L82-L83

My strong preference would be to undo the shell override instead and leave the default of `set -e` in place.

Reply:
Wanna take a crack at this @matthiasdiener?

Reply:
Successfully failed. :tada:

Reply:
its rare to be so happy about a failing CI run :-D